package.json

```diff
@@ -1,40 +1,19 @@
 {
-  "name": "@authless/core",
-  "version": "1.0.2",
-  "description": "A browserless authentication and service management framework",
+  "name": "authless-core",
+  "version": "2.0.0",
+  "description": "",
+  "private": true,
```

Comment:
Except for `version`, I don't understand why any of the other values have been changed. Don't think that is necessary and needs to be changed back before this PR can get merged.